src/coreclr/gc/gc.cpp

```diff
@@ -24119,19 +24325,28 @@ void gc_heap::mark_phase (int condemned_gen_number, BOOL mark_only_p)
     if (gc_t_join.joined())
 #endif //MULTIPLE_HEAPS
     {
-#ifdef MULTIPLE_HEAPS
+#ifdef FEATURE_EVENT_TRACE
+        record_mark_time (gc_time_info[time_plan - 1], current_mark_time, last_mark_time);
+        gc_time_info[time_plan] = last_mark_time;
+#endif //FEATURE_EVENT_TRACE
+
         //decide on promotion
         if (!settings.promotion)
         {
             size_t m = 0;
             for (int n = 0; n <= condemned_gen_number;n++)
             {
+#ifdef MULTIPLE_HEAPS
                 m +=  (size_t)(dd_min_size (dynamic_data_of (n))*(n+1)*0.1);
+#else
+                m +=  (size_t)(dd_min_size (dynamic_data_of (n))*(n+1)*0.06);
```

Comment:
What is the reason the constant is different between SVR and WKS?

Nit: As the rest of the formula is identical, perhaps it would be best to define a constant with a meaningful name that is different between SVR and WKS, and have the computation itself not #ifdef'd.

Reply:
Oh, I see, we already had this different constant below, it's just that you folded the WKS copy of the code.